assets/js/edit.js

```diff
@@ -275,25 +377,105 @@ class Edit extends Component {
 						</PanelRow>
 					</PanelBody>
 				</InspectorControls>
-				<div className={className}>
+				<div className="wp-block-podcasting-podcast-outer">
 					{src ? (
-						<figure key="audio" className={className}>
-							<audio controls="controls" src={src} />
-							{((caption && caption.length) || !!isSelected) && (
-								<RichText
-									tagName="figcaption"
-									placeholder={__(
-										'Write caption…',
-										'simple-podcasting'
+						<>
+							<div className="wp-block-podcasting-podcast__container">
+								{showImage && displayArt && (
+									<div className="wp-block-podcasting-podcast__show-art">
+										<div className="wp-block-podcasting-podcast__image">
+											<img
+												src={showImage}
+												alt={showName}
+											/>
+										</div>
+									</div>
+								)}
+
+								<div className="wp-block-podcasting-podcast__details">
+
+									{displayEpisodeTitle && (
+										<h3 className="wp-block-podcasting-podcast__show-title">
+											{displayEpisodeNumber && (
+												<span>
+													{episodeNumber}.
+												</span>
+											)}
+											{postDetails.title}
+										</h3>
 									)}
-									value={caption}
-									onChange={(value) =>
-										setAttributes({ caption: value })
-									}
-									isSelected={isSelected}
-								/>
-							)}
-						</figure>
+
+									<div className="wp-block-podcasting-podcast__show-details">
+										{displayShowTitle && (
+											<span className="wp-block-podcasting-podcast__title">
+												{showName}
+											</span>
+										)}
+										{displaySeasonNumber && (
```

Comment:
```suggestion
										{displaySeasonNumber && seasonNumber && (
```

HIde this if the season number isn't filled out.

Reply:
@peterwilsoncc Thanks for the suggestion, Resolved.